### DISCUSSION TITLE
Additional Censorship

### DISCUSSION THREAD

**Discussion by ericaVoss** [EHDEN/CatalogueExport#35] [Ideas] [answered]
Example:
406 - Distribution of age by condition_concept_id, condition_concept_id and gender_concept_id

It is possible for there to be 100 record but all the values are the same.  So I saw an example where the age was ALWAYS 24.  So in this case it is probably a case where all 100 records are the same person.  

What are the thoughts about excluding some of the analysis if MIN == MAX?

**Comment by MaximMoinat** [comment, marked as answer]
I am afraid that this is an artifact of the rounding up, using the default small cell count of 5. If there is a single record for that concept_id and gender combination, then this count of 1 is rounded up to 100, which passes the small cell count of 5. Then, all the age quartiles are the same, namely the age of the one person.

Changing the default value of the small cell count prevents this from happening: https://github.com/EHDEN/CatalogueExport/issues/37
